src/coreclr/jit/emitriscv64.cpp

```diff
@@ -2854,7 +2855,12 @@ size_t emitter::emitOutputInstr(insGroup* ig, instrDesc* id, BYTE** dp)
             assert(!"JitBreakEmitOutputInstr reached");
         }
     }
-#endif
+#else  // !DEBUG
+    if (emitComp->opts.disAsm)
+    {
+        emitDispIns(id, false, false, true, 0, *dp, (dstRW - odstRW), ig);
+    }
+#endif // !DEBUG
```

Comment:
Nit: the comment for #endif should reflect the #if, not #else condition.

Ditto elsewhere.